### PR TITLE
add GitHub actions for debugger issues

### PR DESCRIPTION
**Pull request by michelleangela**
Added following GitHub actions to manage debugger issues:
- Close issues with "more info needed" after 14 days of inactivity and involves "andrew" or "pierson"
- Close issues with "questions" after 14 days of inactivity and involves "andrew" or "pierson"
- Close issues with "external" or "by design" immediately
- Close issues with "investigate" or "investigate: costing" after 180 days of inactivity

These actions are defined in separate yml workflows.

### CODE REVIEW

**Review by Colengms — APPROVED** [review]

**Review by Colengms — CHANGES_REQUESTED** [review]
If added a param to StaleCloser.ts, I think you'll need to also add that param to action.yml and update index.ts to pass the value through.

**Comment by michelleangela** [comment]
@WardenGnaw when these are checked in, I will run these new actions manually to test but without committing to any issues.

**Review by Colengms — APPROVED** [review]

File: .github/workflows/question-closer-debugger.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+      - name: Install Actions
+        run: cd ./.github/actions && npm install --production && cd ../..
+      - name: Stale Closer
+        uses: ./.github/actions/StaleCloser
+        with:
+          readonly: ${{ github.event.inputs.readonly }}
+          labels: question, debugger
+          ignoreLabels: language service, internal
```
Comment (michelleangela) [inline .github/workflows/question-closer-debugger.yml:24]: @WardenGnaw this query returns an issue with labels "question" and "debugger" plus other labels the issue may additionally have like "help wanted". Would you like to ignore such issues or auto close them too?

Comment (WardenGnaw) [reply .github/workflows/question-closer-debugger.yml:24]: This query should be good enough for now. I can update it if we run into issues in the future.

File: .github/workflows/more-info-needed-closer-debugger.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        uses: actions/checkout@v2
+      - name: Install Actions
+        run: cd ./.github/actions && npm install --production && cd ../..
+      - name: Stale Closer
+        uses: ./.github/actions/StaleCloser
+        with:
+          readonly: ${{ github.event.inputs.readonly }}
+          labels: more info needed, debugger
+          ignoreLabels: language service, internal
+          involves: wardengnaw, pieandcakes
```
Comment (WardenGnaw) [inline .github/workflows/more-info-needed-closer-debugger.yml]: `+ calgagi`

Calvin also needs to be added to this list.

**Review by WardenGnaw — APPROVED** [review]

File: .github/workflows/more-info-needed-closer-debugger.yml (hunk 2 above)
Comment (michelleangela) [reply .github/workflows/more-info-needed-closer-debugger.yml]: I added `calgagi` to the "question" query issues too.

**Pull request merged**